pages/index.py

```diff
@@ -27,35 +27,59 @@ def slider_item(title, text, active=""):
             slider_item(_("Share, Chat, Browse. Anonymously."), _("""
 Freenet is a platform for censorship-resistant communication and publishing.
 It helps you to remain anonymous, and communicate without fear.
-"""), "active"),
+""") + "\n\n" + """<span class="hidden-xs">""" + _("""
+*'Daddy, where were you when they took freedom of the press away from the Internet?'*
+— Mike Godwin ([read more](about.html))
+""") + """</span>""", "active"),
             slider_item(_("Host a Website"), _("""
 Need a website nobody can take over? That is hosted for free? That is very
 resistant to attacks? Publish it on Freenet!
-""")),
+""") + "\n\n" + """<span class="visible-xs">""" + _("""
+*'I never had to worry no more about security of my sites.'*
```

Comment:
I’m not sure; I like it … but I’m no native speaker, so grammar does not hurt as much for me. Maybe use the other quote for visible-xs instead?